src/extensionPoints/triggerExtension.ts

```diff
@@ -418,6 +428,37 @@ export abstract class TriggerExtensionPoint extends ExtensionPoint<TriggerConfig
     }
   }
 
+  private attachInitializeTrigger(
+    $element: JQuery<Document | HTMLElement>
+  ): void {
+    this.initializeObserver?.disconnect();
+    this.initializeObserver = null;
+
+    // The caller will have already waited for the element. So $element will contain at least one element
+    if (this.attachMode === "once") {
+      for (const element of $element.get()) {
+        void this.runTrigger(element);
+      }
+
+      return;
+    }
+
+    this.initializeObserver = initialize(
+      this.triggerSelector,
+      (index, element) => {
+        void this.runTrigger(element as HTMLElement).then((errors) => {
```

Comment:
Nit: We can use a regular async function here
```suggestion
      async (index, element) => {
        const errors = await this.runTrigger(element as HTMLElement);
```